src/engraving/utests/tempomap_data/absolute_tempo_80_to_120_bpm/absolute_tempo_80_to_120_bpm.mscx

```diff
@@ -0,0 +1,396 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<museScore version="4.00">
+  <programVersion>4.0.0</programVersion>
```

Comment:
test files used to not include programVersion information if you generated them when running musescore with -t, not sure if including this information in these files is now intentional?

Reply:
yep, it's absolutely intentional thing nowadays